examples/llama2/llama2_multilora.ipynb

```diff
@@ -0,0 +1,355 @@
+{
```

Comment:
<p>Could you help add a link to tell the complete configs for <code>export_adapters</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/Olive/pull/1114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Since I have the olive cli work pending I was planning at some docs and info for this script that time. Didn't want to do some work here which would get changed soon https://github.com/microsoft/Olive/tree/jambayk/olive-cli